solvers/saba.py

```diff
@@ -7,35 +7,28 @@
     import numpy as np
     from numba import njit, prange
     from numba.experimental import jitclass
-    constants = import_ctx.import_from('constants')
-    MinibatchSampler = import_ctx.import_from(
-        'minibatch_sampler', 'MinibatchSampler'
-    )
-    spec_minibatch_sampler = import_ctx.import_from(
-        'minibatch_sampler', 'spec'
-    )
-    LearningRateScheduler = import_ctx.import_from(
-        'learning_rate_scheduler', 'LearningRateScheduler'
-    )
-    spec_scheduler = import_ctx.import_from(
-        'learning_rate_scheduler', 'spec'
-    )
+
+    from benchmark_utils import constants
+    from benchmark_utils.minibatch_sampler import MinibatchSampler
+    from benchmark_utils.minibatch_sampler import spec as mbs_spec
+    from benchmark_utils.learning_rate_scheduler import LearningRateScheduler
+    from benchmark_utils.learning_rate_scheduler import spec as sched_spec
 
 
 class Solver(BaseSolver):
     """Stochastic Average Bi-level Algorithm."""
     name = 'SABA'
 
     stopping_criterion = SufficientProgressCriterion(
-        patience=constants.PATIENCE, strategy='callback'
+        patience=12_800, strategy='callback'
```

Comment:
As we discussed, we should use smaller `patience` and keep the same for all solvers
```suggestion
        patience=100, strategy='callback'
```